addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -419,6 +419,10 @@ export function goToTheme(position = "bottom") {
             tooltipPosition: position,
             run: "click",
         },
+        {
+            content: "Check that the theme tab is active",
+            trigger: ".o-tab-content .options-container [data-action-id='switchTheme']",
+        },
```

Comment:
The "homepage" tours are supposed to be tours that are also shown to the users, for them to discover the builder (first tour launched after the configurator). Now... it has been known to have issues / be outdated (hence what the configurator loading screen is currently hiding the "follow the tour tip" message if I am not mistaken). But let's not pile on the problem: that step is not supposed to be shown to users... and I am not sure how it works in the new system of tours. I was thinking there was a `isActive` key or something like that, but maybe it was changed to be automatic for steps that have no `run`? (like the first step here actually). Could you confirm me that? :pray: